__tests__/app.test.js

```diff
@@ -113,3 +113,41 @@ describe("GET /api/articles", () => {
       });
   });
 });
+describe("GET /api/articles/:article:id/comments", () => {
+  test("get articles by id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        comments.rows.forEach((comments) => {
+          expect(comments).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number),
```

Comment:
I'm not sure we should expect `article_id `to match `expect.any(Number),` - surely it should be 1?